gemd/util/impl.py

```diff
@@ -105,7 +105,7 @@ def substitute_objects(obj, index):
                        applies=lambda o: isinstance(o, LinkByUID))
 
 
-def flatten(obj):
+def flatten(obj, scope):
```

Comment:
This argument order can be swapped if we want; I think we were discussing forcing named arguments in general instead.